app.R

```diff
@@ -158,6 +161,15 @@ server <- function(session, input, output) {
       current.run(createAppDT(all.runs.dt()[name == input$run, path]))
       all.studies <- current.run()$Study
       updateSelectInput(session, "study", choices = all.studies)
+      
+      # Add links that lead to useful places
+      output$notificationMenu <- renderMenu({
+        dropdownMenu(
+          notificationItem("SeqWare Run Report", status = "info", icon = icon("copy"), href = generateRunReportURL(input$run)),
+          notificationItem("MISO Run Report", status = "info", icon = icon("cog"), href = generateMisoRunURL(input$run)),
```

Comment:
Are you strongly attached to the "copy" and "cog" icons? Many web pages use icons very similar to these to indicate "copy" and "settings", which is not what they're being used for here. What do you think about something like [extrnal-link-alt](https://fontawesome.com/icons/external-link-alt?style=solid) for both of them?

Reply:
For unknown reasons, that icon does not display. What about below:
![image](https://user-images.githubusercontent.com/37381890/38938796-6bd3a72a-42f4-11e8-9ab6-b82c151b1f0b.png)

Reply:
It [looks like it should work if you specify](https://shiny.rstudio.com/reference/shiny/1.0.1/icon.html) `icon = icon("external-link-alt", lib = "font-awesome")` but I have no strong preference for the external link icon over the link icon you've got here.
Could the "Dropdown menu" link icon be changed to be the same icon as the notification item link icons?

Reply:
Still did not work, so I am following link icon strategy. Looks like this:
![image](https://user-images.githubusercontent.com/37381890/38945392-67c6be1e-4304-11e8-907d-4e189264c04e.png)